Source/Core/Core/HW/SI/SI_DeviceGBA.cpp

```diff
@@ -136,39 +128,29 @@ void GBAConnectionWaiter_Shutdown()
     s_connection_thread.join();
 }
 
-static bool GetAvailableSock(std::unique_ptr<sf::TcpSocket>& sock_to_fill)
+template <typename T>
+static std::unique_ptr<T> MoveFromFront(std::queue<std::unique_ptr<T>>& ptrs)
```

Comment:
I didn't find anything. Apparently the reasoning behind it's absence is to not violate invariants when using exceptions: http://stackoverflow.com/questions/25035691/why-doesnt-stdqueuepop-return-value